requirements-other.txt

```diff
@@ -0,0 +1,2 @@
+pickle5
+python-igraph
```

Comment:
And actually, now that I look at this - do we really want to support this? I've never used it

Reply:
Let's rename the PR, add pickle to the main requirement file, and delete python-igraph

Reply:
And whatever function is hiding somewhere that uses it

Reply:
Grep doe snot show any utilization. PR updated.

Reply:
GCNN/data_utils/tester.py

Reply:
Need to address/remove this file before merging ^